### ISSUE TITLE
Failed Test PR#286 - Failing endpoint /evm/approve for mad-meerkat/vvs

### ISSUE THREAD

**Issue by Deepali6997** [CoinAlpha/hummingbot#288]
**Summary and Screenshots**
When running the following curl script, it throws the error as shown below

<img width="1728" alt="Screenshot 2022-10-21 at 9 04 36 PM" src="https://user-images.githubusercontent.com/100112937/197233886-da670776-85b6-427a-8671-9effc9b5bdfb.png">

Gateway Logs : 
```
2022-10-22 04:27:58 | error | 	The spender param is not a valid Ethereum address (0x followed by 40 hexidecimal characters). | Error: The spender param is not a valid Ethereum address (0x followed by 40 hexidecimal characters).
    at throwIfErrorsExist (/Users/deepaligautam/PR286/gateway/dist/src/services/validators.js:38:15)
    at /Users/deepaligautam/PR286/gateway/dist/src/services/validators.js:85:40
    at /Users/deepaligautam/PR286/gateway/dist/src/evm/evm.routes.js:37:58
    at Generator.next (<anonymous>)
    at /Users/deepaligautam/PR286/gateway/dist/src/evm/evm.routes.js:8:71
    at new Promise (<anonymous>)
    at __awaiter (/Users/deepaligautam/PR286/gateway/dist/src/evm/evm.routes.js:4:12)
    at /Users/deepaligautam/PR286/gateway/dist/src/evm/evm.routes.js:36:87
    at /Users/deepaligautam/PR286/gateway/dist/src/services/error-handler.js:37:28
    at Layer.handle [as handle_request] (/Users/deepaligautam/PR286/gateway/node_modules/express/lib/router/layer.js:95:5)
    at next (/Users/deepaligautam/PR286/gateway/node_modules/express/lib/router/route.js:144:13)
    at Route.dispatch (/Users/deepaligautam/PR286/gateway/node_modules/express/lib/router/route.js:114:3)
    at Layer.handle [as handle_request] (/Users/deepaligautam/PR286/gateway/node_modules/express/lib/router/layer.js:95:5)
    at /Users/deepaligautam/PR286/gateway/node_modules/express/lib/router/index.js:284:15
    at Function.process_params (/Users/deepaligautam/PR286/gateway/node_modules/express/lib/router/index.js:346:12)
    at next (/Users/deepaligautam/PR286/gateway/node_modules/express/lib/router/index.js:280:10)
```

/evm/approve
`curl -s -X POST -k --key $GATEWAY_KEY --cert $GATEWAY_CERT -H "Content-Type: application/json" -d "$(envsubst < ./requests/cronos_approve.json)" https://localhost:15888/evm/approve | jq`

**Steps To Reproduce**

Here are the steps to reproduce the issue 
1.  After you're done starting the gateway from the source open a separate terminal
2. open separate terminal then export environment needed (CRONOS_ADDRESS, CRONOS_PRIVATE_KEY, GATEWAY_CERT and GATEWAY_KEY)
3. send request using curl
4. observe the error happen

**Test Result Comparison**
Not available on the master and dev branches of the foundation yet

**Attachments**
gateway log : 
[logs_gateway_app.2022-10-21.log](https://github.com/CoinAlpha/hummingbot/files/9840471/logs_gateway_app.2022-10-21.log)

**Comment by PtrckM** [comment]
Adding additional info:
- added `CRONOS_CONNECTOR='mad_meerkat'`, however still getting error messages:
  - `Network error. Please check your node URL, API key, and Internet connection.`
  - `failed to deduct transaction costs from user balance: the tx gasfeecap is lower than the tx baseFee: 100000000000 (gasfeecap), 4892618666104 (basefee) : insufficient fee: insufficient fee`
- also tried `CRONOS_CONNECTOR='vvs'` same issue/error.


![image](https://user-images.githubusercontent.com/54516858/197318840-700f1e9a-024e-4ccc-8a67-9f2911705774.png)

```
{
  "message": "Network error. Please check your node URL, API key, and Internet connection.",
  "httpErrorCode": 503,
  "errorCode": 1001,
  "stack": "Error: processing response error (body=\"{\\\"jsonrpc\\\":\\\"2.0\\\",\\\"id\\\":300,\\\"error\\\":{\\\"code\\\":-32000,\\\"message\\\":\\\"failed to deduct transaction costs from user balance: the tx gasfeecap is lower than the tx baseFee: 100000000000 (gasfeecap), 4892618666104 (basefee) : insufficient fee: insufficient fee\\\"}}\\n\", error={\"code\":-32000}, requestBody=\"{\\\"method\\\":\\\"eth_sendRawTransaction\\\",\\\"params\\\":[\\\"0x02f8b31981db85174876e80085174876e800832dc6c094c21223249ca28397b4b6541dffaecc539bff0c5980b844095ea7b3000000000000000000000000145677fc4d9b8f19b5d56d1820c48e0443049a30ffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffc080a0306a04e74227d96fad276c53009b4a0fe45b774e76c659737236b6f2f6362478a02c8130779213f3cbd4590809a91ef6f1fc1c3c6ff9a98b14ed2667dc3a5f0362\\\"],\\\"id\\\":300,\\\"jsonrpc\\\":\\\"2.0\\\"}\", requestMethod=\"POST\", url=\"https://evm.cronos.org/\", code=SERVER_ERROR, version=web/5.6.1)\n    at Logger.makeError (/usr/src/app/node_modules/@ethersproject/logger/lib/index.js:233:21)\n    at Logger.throwError (/usr/src/app/node_modules/@ethersproject/logger/lib/index.js:242:20)\n    at /usr/src/app/node_modules/@ethersproject/web/lib/index.js:305:32\n    at step (/usr/src/app/node_modules/@ethersproject/web/lib/index.js:33:23)\n    at Object.next (/usr/src/app/node_modules/@ethersproject/web/lib/index.js:14:53)\n    at fulfilled (/usr/src/app/node_modules/@ethersproject/web/lib/index.js:5:58)\n    at processTicksAndRejections (node:internal/process/task_queues:96:5)"
}

```

**Comment by Deepali6997** [comment]
now i am also getting the same response.

**Comment by PtrckM** [comment]
- confirmed fix using commit 343be8445f82ec94e20262439fbe0e8e95cb8954 for either mmf or vvs

![image](https://user-images.githubusercontent.com/54516858/197710187-c53d59e7-8e2e-4504-afc8-29e7c2ad401c.png)

![image](https://user-images.githubusercontent.com/54516858/197710501-23721c19-e0c3-4e16-abd5-c5efdbbed42f.png)
